element/index.js

```diff
@@ -80,9 +86,16 @@ export { createPortal };
  *
  * @param {WPElement} element Element to render
  *
- * @returns {String} HTML.
+ * @returns {string} HTML.
  */
-export { renderToStaticMarkup as renderToString };
+export function renderToString( element ) {
+	let rendered = renderToStaticMarkup( element );
+
+	// Drop raw HTML wrappers (support dangerous inner HTML without wrapper)
+	rendered = rendered.replace( /<\/?wp-dangerous-html>/g, '' );
```

Comment:
Nit: this function can return here instead of updating variable's value.

Reply:
>Nit: this function can return here instead of updating variable's value.

I'm going to leave this as-is, as it distinguishes / sets a pattern for transformations which would be applied to the default rendered output.